.github/workflows/gradlew-update.yaml

```diff
@@ -7,11 +7,11 @@ on:
 jobs:
   update:
     runs-on: ubuntu-22.04
-    if: github.repository == 'jellyfin/jellyfin-sdk-kotlin'
+    if: ${{ contains(github.repository_owner, 'jellyfin') }}
```

Comment:
Why contains instead of equals?

Reply:
We match by organization so we can easily copy workflows between repositories without changing too much. This specific line is copied from @h1dden-da3m0n's workflows. I believe we use contains so it still works for the jellyfin-archive and jellyfin-sandbox organizations.